src/main/java/org/bonitasoft/engine/connector/uipath/UIPathService.java

```diff
@@ -17,22 +17,10 @@
 import java.util.List;
 import java.util.Map;
 
-import org.bonitasoft.engine.connector.uipath.model.AddToQueueRequest;
-import org.bonitasoft.engine.connector.uipath.model.CloudAuthentication;
-import org.bonitasoft.engine.connector.uipath.model.Job;
-import org.bonitasoft.engine.connector.uipath.model.JobRequest;
-import org.bonitasoft.engine.connector.uipath.model.QueueItem;
-import org.bonitasoft.engine.connector.uipath.model.Release;
-import org.bonitasoft.engine.connector.uipath.model.Robot;
+import org.bonitasoft.engine.connector.uipath.model.*;
```

Comment:
Please do not use wild card in your import statement, it's a bad practice and can lead to issues.
-> https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
this has been updated